A21-Service-Config-Error-Handling.md

```diff
@@ -50,8 +53,8 @@ configuration.
 	config. An empty service config is valid.
 	2. If the client has not previously selected a valid service config or no
 	service config, for example a new client, then:
-		1. It uses the default service config, if configured. Note that, an
-		empty default service config is a valid service config.
+		1. It uses the default service config, if configured and valid. Note
+		that, an empty default service config is a valid service config.
```

Comment:
Grammar nit: The comma should be removed here.

Reply:
Removed!